Tests/test_card_animation.gd

```diff
@@ -0,0 +1,65 @@
+extends TestBase
+
+
+var created_cast_nodes: Array[Variant] = []
+
+
+# @Override
+func before_each() -> void:
+	super()
+	CardManager.disable_card_animations = false
+
+
+func after_each() -> void:
+	super()
+	created_cast_nodes.clear()
+
+
+func test_player_animation_created() -> void:
+	var card: CardBase = load("res://Cards/Resource/Card_Damage.tres").duplicate()
+	var cast_anim: CastAnimationData = CastAnimationData.new()
+	cast_anim.cast_animation_scene = load("res://Cards/Animation/Anim_Slap.tscn")
+	cast_anim.cast_position = CastPos_Caster.new()
+	card.card_effects_data[0].animation_data = cast_anim
+	
+	card.on_card_play(_player, _enemy)
```

Comment:
is there a reason for playing the card ? isn't the animation already supposed to play the card after it's done ?

Reply:
No, calling `on_card_play` is what triggers the animation

Reply:
oh I see